src/components/MentorsList/MentorsList.js

```diff
@@ -43,31 +43,27 @@ export default class MentorsLists extends Component {
     const mentorsInList = mentors.slice(0, page * itemsInPage);
 
     return (
-      <div className={classNames([
-        'mentors-wrapper',
-        className
-      ])}>
+      <section className={classNames(["mentors-wrapper", className])}>
```

Comment:
No need to use `classNames()`. How do you feel about using:
```<section className={`mentors-wrapper ${className}`}>```? 😃

Reply:
If `className` is empty, there will be no extra space 😉.

Reply:
Also if `className` is undefined, you will have a `undefined` class in the DOM:

```
<section class="mentors-wrapper undefined">
```

Which is not ideal